cluster.go

```diff
@@ -35,20 +36,27 @@ func NewClusterClient(opt *ClusterOptions) *ClusterClient {
 	}
 	client.commandable.process = client.process
 	client.reloadIfDue()
-	go client.reaper(time.NewTicker(5 * time.Minute))
+	go client.reaper()
 	return client
 }
 
-// Close closes the cluster client.
+// Close closes the cluster client, releasing any open resources.
+//
+// It is rare to Close a Client, as the Client is meant to be
+// long-lived and shared between many goroutines.
 func (c *ClusterClient) Close() error {
-	// TODO: close should make client unusable
-	c.setSlots(nil)
+	defer c.clientsMx.Unlock()
```

Comment:
It should be slightly faster (I believe), because `defer c.clientsMx.Unlock()` is evaluated before the lock is acquired. It does not matter too much and I have to stop doing this.